packages/block-editor/src/components/block-mover/index.js

```diff
@@ -79,8 +115,9 @@ export class BlockMover extends Component {
 				<IconButton
 					className="editor-block-mover__control block-editor-block-mover__control"
 					onClick={ isLast ? null : onMoveDown }
-					icon={ downArrow }
-					label={ __( 'Move down' ) }
+					icon={ getArrowIcon( 'right' ) }
+					// translators: %s: Horizontal direction of block movement ( left, right )
+					label={ orientation === 'horizontal' ? sprintf( __( 'Move %s' ), getMovementDirection( 'right' ) ) : __( 'Move down' ) }
```

Comment:
Same again here about moving the `orientation === 'horizontal'` check into `getMovementDirection`.